ksqldb-common/src/main/java/io/confluent/ksql/schema/ksql/SchemaConverters.java

```diff
@@ -423,6 +426,7 @@ public SqlType toSqlType(final ParamType paramType) {
             .put(ParamTypes.DOUBLE, SqlBaseType.DOUBLE)
             .put(ParamTypes.DECIMAL, SqlBaseType.DECIMAL)
             .put(ParamTypes.TIMESTAMP, SqlBaseType.TIMESTAMP)
+            .put(ParamTypes.LAMBDALITERAL, SqlBaseType.LAMBDA)
```

Comment:
this is error prone because any `LambdaType` that checks with this map that isn't _exactly_ `Lambda.of(null, null)` will not match and will throw an exception.